dataplane/standalone/translator.h

```diff
@@ -132,8 +114,10 @@ class Translator {
                                   sai_status_t *object_statuses);
 
  private:
+  std::shared_ptr<AttributeManager> attrMgr;
   std::shared_ptr<forwarding::Forwarding::Stub> client;
-  std::unordered_map<sai_object_id_t, SaiObject> objects;
+  std::unordered_map<sai_object_id_t, std::shared_ptr<Switch>> switches;
+  std::unordered_map<std::string, std::shared_ptr<APIBase>> apis;
```

Comment:
Doesn't the switch also contain the object apis (IIUC)? Are there apis that are non-switch apis?

Reply:
i believed there are, add todo to confirm